app/views/locations/_shipping_office.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="shipping-office" data-latitude="<%= shipping_office.latitude %>" data-longitude="<%= shipping_office.longitude %>" data-name="<%= shipping_office.name %>" id="office-<%= shipping_office.id %>">
+<div class="shipping-office" id="office-<%= shipping_office.id %>">
```

Comment:
These `data-*` attributes weren't being used anywhere in the UI and _may_ have been causing another hit to the database, so… _see ya later_.